### PR TITLE
CI: don't include debug symbols for musl builds

### PR DESCRIPTION
**Pull request by julianoes**
When we add the debug symbols for the musl builds we end up with binaries around 300 to 400 MB which are too big for PyPi.

### CODE REVIEW

**Review by JonasVautherin — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ jobs:
       - name: install examples dependencies
         run: sudo apt-get install libsdl2-dev
       - name: configure examples
-        run: cmake -DCMAKE_PREFIX_PATH=$(pwd)/build/release/third_party/install -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWERROR=OFF -Bexamples/build -Hexamples
+        run: cmake -DCMAKE_PREFIX_PATH=$(pwd)/install -DCMAKE_PREFIX_PATH=$(pwd)/build/release/third_party/install -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWERROR=OFF -Bexamples/build -Hexamples
```
Comment (JonasVautherin) [inline .github/workflows/main.yml:107]: Are you sure this works? I think I usually do something more like `-DCMAKE_PREFIX_PATH="$(pwd)/install;$(pwd)/build/release/third_party/install"` :thinking:

Comment (julianoes) [reply .github/workflows/main.yml:107]: Oh, I think you're right!

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -104,7 +104,7 @@ jobs:
       - name: install examples dependencies
         run: sudo apt-get install libsdl2-dev
       - name: configure examples
-        run: cmake -DCMAKE_PREFIX_PATH=$(pwd)/build/release/third_party/install -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWERROR=OFF -Bexamples/build -Hexamples
+        run: cmake -DCMAKE_PREFIX_PATH=$(pwd)/install;$(pwd)/build/release/third_party/install -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWERROR=OFF -Bexamples/build -Hexamples
```
Comment (JonasVautherin) [inline .github/workflows/main.yml:107]: I would use double quotes, because otherwise I am not sure how the `;` is interpreted.

Comment (JonasVautherin) [reply .github/workflows/main.yml]: Yep, seems like that was it :blush:.

**Review by JonasVautherin — DISMISSED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (julianoes) [reply .github/workflows/main.yml]: Thanks!

**Pull request merged**